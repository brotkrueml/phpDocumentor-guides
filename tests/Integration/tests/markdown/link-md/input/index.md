# Markdown with links

This is a Markdown document with some basic formatting. 

See also: [Link Text](https://www.example.com) 