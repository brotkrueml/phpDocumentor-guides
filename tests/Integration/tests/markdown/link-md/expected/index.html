<!DOCTYPE html>
<html lang="en">
<head>
    <title>Markdown with links - Project Title</title>

</head>
<body>
    <h1>Markdown with links</h1>

<p>This is a Markdown document with some basic formatting.</p>
<p>See also: <a href="https://www.example.com">Link Text</a></p>
</body>
</html>
